src/edit/index.vue

```diff
@@ -82,11 +91,18 @@ export default {
             fileName: null
           });
         });
+    },
+    reload() {
+      const { playable } = this.element.data;
+      if (playable) this.$emit('save', this.element.data);
     }
   },
   watch: {
     'element.data.uploadUrl'() {
       if (this.isReadyToUpload) this.upload();
+    },
+    'isUnfocus'(newValue, oldValue) {
+      if (newValue && !oldValue) this.reload();
```

Comment:
Do we have to check the old value here, since it's a primitive value? (I'd assume watch effect won't be invoked if the primitive value doesn't really change **value**)

Reply:
Because otherwise this will trigger reload in the case when an element is not selected at first and then selected. We want to reload the element only when it was selected and then unselected. Am I right?

Reply:
Maybe I should alias it with `hasLostFocus` variable so it's more readable. What do you think?

Reply:
```js
const hasLostFocus = newValue && !oldValue;
if (hasLostFocus) this.reload();
```